.gitignore

```diff
@@ -3,3 +3,4 @@ subprojects/*/
 docs/html/
 .vscode/settings.json
 .vscode/cspell.json
+unit_test_files
```

Comment:
I happen to run the unit tests in the root of the working copy. But if we really want to debate every entry in .gitignore, it might be a better solution to remove it from the repo entirely and let everyone have their own copy. I'll add a slash for now.

Reply:
> But if we really want to debate every entry in .gitignore, it might be a better solution to remove it

Well, obviously the entry was not what has been intended. You want to ignore the directory even in the root dir (answering my question what it is needed for), but what you did was ignoring the directory and all also files with that name.

Is a review not to point out where specified / written-out things do not match intended things?

And yes, I think every entry in `.gitignore` should be open to discussion as every other line added to any other file.

Writing it without a slash implies some other use.
* I ask what your intention is
* I report what my usecase would be
* I suggest a fix

:woman_shrugging: